R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,96 @@
+# importCNA.GISTIC2.R
```

Comment:
I'm entering some comments here - of course coding this is secondary to coding the tests. Just some stuff I noticed.